src/signals/incident-management/containers/IncidentDetail/components/Attachments/Attachments.test.tsx

```diff
@@ -702,8 +704,10 @@ describe('Attachments', () => {
           />
         )
       )
-      const removeButton = screen.getByTitle(/bijlage verwijderen/i)
-      userEvent.click(removeButton)
+      const deleteButton = screen.getByTestId(
+        'https://ae70d54aca324d0480ca01934240c78f.jpg2020-06-10T11:51:24.281272+02:00'
```

Comment:
huh??

Reply:
screen.getByTitle(/bijlage verwijderen/i) will give you an array. You need to target an individual icon to click. I've changed the name to be slightly more readable.